spec.html

```diff
@@ -4002,7 +4002,7 @@
         1. If the _LeftFirst_ flag is *true*, then
           1. Let _px_ be ? ToPrimitive(_x_, hint Number).
           1. Let _py_ be ? ToPrimitive(_y_, hint Number).
-        1. Else the order of evaluation needs to be reversed to preserve left to right evaluation
+        1. Else the order of evaluation needs to be reversed to preserve left to right evaluation,
```

Comment:
I feel like this is not actually a condition but an assert and so would argue weakly for "Else, the order of evaluation..." instead. But this is fine.